__tests__/app.test.js

```diff
@@ -73,13 +73,49 @@ describe.only("GET /api/reviews/:review_id", () => {
         });
       });
   });
-  test("400: responds when review at given id doesnt not exist", () => {
+  test("404: responds when at given id doesnt not exist", () => {
     return request(app)
       .get("/api/reviews/200")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("No review found for review_id: 200");
+      });
+  });
+  test("400: responds when at given an invalid Id", () => {
+    return request(app)
+      .get("/api/reviews/banana")
       .expect(400)
       .then(({ body }) => {
         const { msg } = body;
-        expect(msg).toBe("Review does not exist");
+        expect(msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe.only("GET /api/users", () => {
+  test("200: responds with a users array of objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        users.forEach((user) => {
```

Comment:
If you are using a forEach, you want to also test for the length of users to make sure that it is running every time and more robust.

Reply:
Hi Cat, I added a new line of code to test this, but i am unsure if this is what you meant by testing for the length